### PR TITLE
feat(ui5-card): add properties loading and loadingDelay

### PR DESCRIPTION
**Pull request by alexandar-mitsev**
Fixes: #9437

**Thank you for your contribution!** 👏

### PR checklist
- [x] Check the [Development Hints](https://sap.github.io/ui5-webcomponents/docs/contributing/DoD/)

- [x] Follow the [Commit message Guidelines](https://github.com/SAP/ui5-webcomponents/blob/main/docs/6-contributing/02-conventions-and-guidelines.md#commit-message-style)

For example: `fix(ui5-*): correct/fix sth` or `feat(ui5-*): add/introduce sth`. If you don't want the change to be part of the release changelog - use `chore`, `refactor` or `docs`.

- [x] Add proper description about the background of the change and the change itself

- [x] Link to an existing issue (if available)

Use `Fixes: {#PR_NUMBER}` to close the issue automatically when the PR is merged
or `Related to: {#PR_NUMBER}` to just create a link between the PR and the issue.

- [x] Read the [Contributing Guidelines](https://github.com/SAP/ui5-webcomponents/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: packages/main/src/Card.ts [hunk 1]
```diff
@@ -85,6 +87,24 @@ class Card extends UI5Element {
 	@slot({ type: HTMLElement, invalidateOnChildChange: true })
 	header!: Array<CardHeader>;
 
+	/**
+	 * Defines if a busy indicator would be displayed over the card.
+	 * @default false
+	 * @public
+	 * @since 2.1.0-rc.1
```
Comment (dimovpetar) [inline packages/main/src/Card.ts:94]: Double check which version will this be released with: 2.1.0-rc.2 or 2.1.0-rc.3

File: packages/main/src/Card.ts [hunk 2]
```diff
@@ -85,6 +87,24 @@ class Card extends UI5Element {
 	@slot({ type: HTMLElement, invalidateOnChildChange: true })
 	header!: Array<CardHeader>;
 
+	/**
+	 * Defines if a busy indicator would be displayed over the card.
```
Comment (dimovpetar) [inline packages/main/src/Card.ts:91]: List and Table use "loading indicator" instead of "busy indicator". I think it will be better to use it here too

**Review by dimovpetar — CHANGES_REQUESTED** [review]

File: packages/main/src/Card.ts (hunk 2 above)
Comment (alexandar-mitsev) [reply packages/main/src/Card.ts]: Done

File: packages/main/src/Card.ts (hunk 1 above)
Comment (alexandar-mitsev) [reply packages/main/src/Card.ts]: Done

**Review by dimovpetar — APPROVED** [review]

**Pull request merged**